plugins/history.rst

```diff
@@ -1,4 +1,20 @@
 History Plugin
 ==============
 
-TODO: explain the error plugin
+The `HistoryPlugin` notifies a Http\Client\Plugin\Journal of all successful and failed calls. As an example,
+HttplugBundleThis use this plugin for collecting responses or exceptions associated with requests::
+
+    use Http\Discovery\HttpClientDiscovery;
+    use Http\Client\Plugin\PluginClient;
+    use Http\Client\Plugin\HistoryPlugin;
+
+    $historyPlugin = new HistoryPlugin(new \My\Journal\Implementation());
+
+    $pluginClient = new PluginClient(
+        HttpClientDiscovery::find(),
+        [$historyPlugin]
+    );
+
+
+This plugin only collect data after resolution, for logging purpose it's best to use the `LoggerPlugin` which logs
```

Comment:
This plugin collects data only after the request resolution. For logging purposes, ...